<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>1920</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>13</height>
     </size>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>274</number>
    </attribute>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата следующего дня</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата предыдущего дня</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Является ли год високосным</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер недели в году</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество дней до дня рождения</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество дней между датами</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="loadFile">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>11</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bithdayLine">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateDate">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить дату</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateDifDate">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить дату</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="difDateLine">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addDate">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>10</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить дату</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addDateLine">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteDate">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>10</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить выбранную дату</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
